keps/sig-instrumentation/1661-event-api-migration/kep.yaml

```diff
@@ -0,0 +1,17 @@
+title: Event API Migration
+authors:
+  - "@chelseychen"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+  - sig-scalability
+  - sig-architecture
+status: provisional
+creation-date: 2020-04-01
+reviewers:
+  - "@yastij"
+  - "@wojtekt"
+approvers:
+  - "@wojtekt"
```

Comment:
we also need an approver from sig-instrumentation

Reply:
I've added tech lead @Frederic Branczyk for instrumentation.